tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rules/ChooseRule.scala

```diff
@@ -36,7 +35,6 @@ class ChooseRule(rewriter: SymbStateRewriter) extends RewritingRule {
       case OperEx(TlaOper.chooseBounded, varName, set, pred) =>
         // This is a general encoding, handling both happy and unhappy scenarios,
         // that is, when CHOOSE is defined on its arguments and not, respectively.
-        def solverAssert = rewriter.solverContext.assertGroundExpr _
```

Comment:
This is curious. Will need to check what the assert does.

Reply:
Oh, I see. This was just giving a short naming to the lambda here, and at some point we stopped calling the function.

Reply:
Yes, it was just a shorthand